resources/lib/kodi/library.py

```diff
@@ -322,6 +354,8 @@ def _add_to_library(videoid, export_filename):
             # it an appropriate videoid for later reference
             library_node[id_item] = {
                 'videoid': videoid.derive_parent(depth)}
+            if depth < 2:  # If Show or Season, save the nfo export status
```

Comment:
I ask for info, instead of depth, it is not possible to verify the videoid value of the node?

Reply:
can be the mediatype of the parent videoid yes, and will avoid remember that for movies.